tests/api/registration.spec.js

```diff
@@ -33,7 +33,7 @@ describe('User registration', () => {
           server.listen(3001, callback)
         },
       ],
-      done,
+      done
```

Comment:
Please run the linter before pushing. Don't worry about trailing commas failing tests, since trailing commas are generally safe. We'll deal with the tests later

Reply:
the problem is not with failing test but the code will just not run in my local machine if i add those commas. it throws error.